java-core/google-cloud-core/clirr-ignored-differences.xml

```diff
@@ -12,4 +12,9 @@
         <className>com/google/cloud/ReadChannel</className>
         <method>long limit()</method>
     </difference>
+    <difference>
+        <differenceType>7012</differenceType>
+        <className>com/google/cloud/testing/BaseEmulatorHelper$EmulatorRunner</className>
+        <method>int waitForDuration(java.time.Duration)</method>
+    </difference>
```

Comment:
This interface method has a `default` implementation

Reply:
Can you add this as a comment to the file? i.e. something like `Default method added to interface`

Reply:
Added a comment.